src/traditional_de/estimate_delay.jl

```diff
@@ -115,6 +120,55 @@ function exponential_decay_fit(X, Y, weight = :equal)
     end
 end
 
+"""
+    average_displacement(x, τ, m) -> S_m(τ)
+Determines the average displacement from the diagonal generalized SSR given a τ.
+Adapted from:  https://www.delucafoundation.org/download/bibliography/de-luca/061.pdf.
```

Comment:
What is SSR? Please expand it fully and avoid abbreviations.

Additionally, please properly cite the actual paper using the syntax that is used in the rest of the docstrings. `[^Rosenstein1993]`.